### ISSUE TITLE
Vue Component is not working when using this package

### ISSUE THREAD

**Issue by Hujjat** [renatomarinho/laravel-page-speed#59]
<p>When I use this package, my Vue Components are not working. </p>
<h2>Detailed description</h2>
<p>It compresses the HTML code but the Vue components are not working. I use ajax request and fetch data to a single file Vue Component. Then I used it on blade page. Everything is working, but when I use this package to compress the code, the Vue component can't read the properties. </p>
<p>It's a bug</p>
<h2>Your environment</h2>
<p>I'm using Windows and Wamp server latest version. </p>
<ul>
<li>Version used (e.g. PHP 7.1, HHVM 3): PHP 7.1, Laravel 5.5</li>
<li>Operating system and version (e.g. Ubuntu 16.04, Windows 7): Windows 10</li>
</ul>

**Comment by vesper8** [comment]
Same here, I use Vue a plenty and it seems to break when inline json is passed as vue component props.

I'm sure @renatomarinho can be fix this. Hope he does soon :) Here's an example line of code that if included in the html will break

```
<div class="form-group" :class="{'has-error': supportForm.errors.has('from')}"></div>
```

it causes this Vue error:

```
- invalid expression: Unexpected token { in

    form-group {'has-error': supportForm.errors.has('from')}

  Raw expression: :class="form-group {'has-error': supportForm.errors.has('from')}"
```

**Comment by vesper8** [comment]
I would like to add that it is the InlineCss middleware that causes this, not the CollapseWhitespace

**Comment by tswestendorp** [comment]
This might have been fixed with #63

**Comment by Hujjat** [comment]
Yes, It's working now. 👍
